Transcript for call 14  #25 (#29)
status-im/pm: notes/meeting14.md

3. Miscellaneous announcements

## Transcript
[Transcript](transcript14.md)

## Call Notes

### Swarm Updates

### Sticker Market
- Mostly done with feature work
- Andrey made a few changes to contract interaction
- Still in process of auditing the contract

### Browser
- Andrey implemented EIP712; goal is to provide a different way to sign data from within your DApp
    - Previous way was purely string based, now more complex information is supported (requested by DApp developers)
- Also working on separating DApps into their own tab
- Working on a new swarm for extensions to move the product out of alpha

### Tribute to Talk/Chat
- Fixed the tool bar and now making it a component
- Finalizing Tribute to Talk chat-side UI (settings already done)
    - Open issues: can we send a personalized message or not with TtT pay wall?
    - Call this week with Ricardo/Richard to discuss contract additions

### DApp Store
- Moving into implementation of Maciej's DApp store designs (web DApp)
- Using an agency in Cape Town
- Version 1 will be a categorized list
- In the meantime, contract will be prepared for audit

### Teller Network
- Finishing the escrow work flow and UI details

> - Created a ___ system which makes it easier to see and test the DApp

Reviewer: ```suggestion
- Created a data seeding system which makes it easier to see and test the DApp
```

- Deploying new version to Rinkeby soon

### DAO
- Analayzing next steps here right now

### Embark
- Fixing bugs, planning to release beta this week and final 4 version next week
- Launching new website next week as well

Ricardo - liquid democracy, token balances, Whisper J (version 6 from version 3) - should make it easy to build something for Status in Ethereum J

Anna - released hot fix last week mainly to "cancel" a pinned mail server (and opt back into automated selection). Also added changes to the bottom navigation in mobile - always visible now.

> Pedro - we're moving from our ad hoc build system where we download all dependencies we need into next package manager system. This has been on for 3-4 weeks now, we're in final stages before merging to develop. Testing soon. Last remaining items are fixes for Linux and Ubuntu 18. No impact on your existing environment, please to test. External developers are finding it easier to build.

Reviewer: ```suggestion
Pedro - we're moving from our ad-hoc build system where we download all dependencies we need to the Nix package manager system. This has been ongoing for 3-4 weeks now, we're in final stages before merging to develop. Testing soon. Last remaining items are fixes for Linux and Ubuntu 18. No impact on your existing environment, please test. External developers are finding it easier to build.
```
